src/protocol/http/org/apache/jmeter/protocol/http/sampler/HTTPHC4Impl.java

```diff
@@ -1037,7 +1044,7 @@ private CloseableHttpClient setupClient(JMeterVariables jMeterVariables, URL url
 
             // Set up proxy details
             if(useProxy) {
-                HttpHost proxy = new HttpHost(proxyHost, proxyPort);
+                HttpHost proxy = new HttpHost(proxyHost, proxyPort, proxyScheme);
```

Comment:
If proxyScheme is empty, then it should be passed as null so that default (HTTP) is used.

Reply:
@pmouawad 
Added above at line 950: proxyScheme = StringUtils.isBlank(proxyScheme) ? null : proxyScheme;